test/test-pgroonga-parameter.rb

```diff
@@ -0,0 +1,44 @@
+require_relative "helpers/sandbox"
+
+class PGroongaParameterTestCase < Test::Unit::TestCase
+  include Helpers::Sandbox
+
+  sub_test_case "pgroonga.log_rotate_threshold_size" do
+    setup do
+      require_groonga_version(14, 0, 7)
+    end
+
+    def additional_configurations
+      <<-CONFIG
+pgroonga.log_rotate_threshold_size = 10
+      CONFIG
+    end
+
+    test "rotated" do
+      before_log_files = Pathname.glob("#{@postgresql.dir}/pgroonga.log*")
+      run_sql("SELECT pgroonga_command('status');")
+      after_log_files = Pathname.glob("#{@postgresql.dir}/pgroonga.log*")
+      assert_true(before_log_files.size < after_log_files.size)
```

Comment:
ローテート済みのファイルが存在するかどうかだけでいいんじゃない？

```suggestion
      assert do
        not Pathname.glob("#{@postgresql.dir}/pgroonga.log.*").empty?
      end
```

Reply:
たしかに。

Reply:
なるほどです！